core/shared/src/main/scala/cats/effect/cps/AsyncAwait.scala

```diff
@@ -35,45 +35,42 @@ import cats.syntax.all._
  * popularised in other programming languages.
  *
  * {{{
- * object dsl extends AsyncAwaitDsl[IO]
  * import dsl._
  *
  * val io: IO[Int] = ???
- * async { await(io) + await(io) }
+ * async[IO] { io.await + io.await }
  * }}}
  *
  * The code is transformed at compile time into a state machine
  * that sequentially calls upon a [[Dispatcher]] every time it reaches
  * an "await" block.
  */
-class AsyncAwaitDsl[F[_]](implicit F: Async[F]) {
+object dsl {
```

Comment:
Should we move the contents to a `cps` package object, so that people just have to import `cats.effect.cps._` ?

Reply:
Yep! Also should just be a regular object, not a package object, since I don't think we actually need to nest any types.